newIDE/app/src/ObjectEditor/ObjectEditorDialog.js

```diff
@@ -200,6 +200,7 @@ const InnerDialog = (props: InnerDialogProps) => {
                     notifyOfChange();
                   }
                 }}
+                ref={field => field && field.focus()} // Focus the name when the dialog is opened.
```

Comment:
Should you use the `autofocus` props for that?

Reply:
didn't know about this prop! nice!